lint.sh

```diff
@@ -47,11 +47,8 @@ check_groovy(){
          return
        }
   extract_groovy_from_jjb
-  # pipeline_steps/*.groovy includes pipeline_steps/NonCPS.groovy, However
-  # NonCPS must be loaded before any scripts that use it, so it's explicitly
-  # included first.
```

Comment:
This removed comment is still applicable right?

Reply:
Not really as all the support scripts are now loaded before the test scripts.